client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -52,15 +52,18 @@ class SiteTools extends Component {
 			showDeleteContent,
 			showDeleteSite,
 			showThemeSetup,
-			showManageConnection,
+			showDisconnectSite,
 		} = this.props;
 
 		const changeAddressLink = `/domains/manage/${ siteSlug }`;
 		const themeSetupLink = `/settings/theme-setup/${ siteSlug }`;
 		const startOverLink = `/settings/start-over/${ siteSlug }`;
 		const deleteSiteLink = `/settings/delete-site/${ siteSlug }`;
-		const manageConnectionLink = `/settings/manage-connection/${ siteSlug }`;
+		const disconnectSiteLinkWhy =
+			`/settings/disconnect-site/why/${ siteSlug }`;
 
+		const disconnectSiteTitle = translate( 'Disconnect your site' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Disconnect Site' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/16945)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).